halyard-core/src/main/java/com/netflix/spinnaker/halyard/core/resource/v1/JarResourceReader.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2017 Google, Inc.
+ * Copyright 2018 Google, Inc.
```

Comment:
I thought this was representing file creation?

Reply:
Good catch